src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
```

Comment:
Ensure that the element with the class 'logo' exists in the DOM before attaching an event listener. If it doesn't exist, the code will throw an error.